omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml

```diff
@@ -1,427 +1,668 @@
+            if (jq.browser.msie) {
+                jq(":checkbox").click(function () {
+                jq(this).change();
+                });
+            }
+
+            jq('[class^="INREnableDisable"]').change(function () {
+                enable_disable_inr(jq(this));
+            });
+
+            jq('[class^="INREnableDisable"]').each(function () {
+                enable_disable_inr(jq(this));
+            });
+
+            });
+        }
+    </script>    
+
+
+    <div class="ai-page-frame">
+        <div class="headers" style="text-align: center;">
+            <h3 style="background: #FFFF7D; padding: 10px;">INTEGRATED MATERNITY REGISTER (HMIS FORM 072)</h3>
+        </div>
+        <fieldset>
+            <legend>Admission Information</legend>
+            <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                <tr>
+                    <td>
+                        <p><label>Date</label></p>
+                        <p>
+                            <encounterDate default="today" disallowMultipleEncountersOnDate="block" />
+
+                            <span style="display: none;">
+
+                                <encounterLocation default="629d78e9-93e5-43b0-ad8a-48313fd99117"
+                                                   order="629d78e9-93e5-43b0-ad8a-48313fd99117" />
+                            </span>
+                        </p>
+                    </td>
+                    <td>
+                        <p><label>IPD Number</label></p>
+                        <p>
+                            <patient field="identifier" identifierTypeId="a6217c17-5012-4514-b9f2-5d02d3d04c44"
+                                     required="true" />
+                        </p>
+                    </td>
+                    <td>
+                        <p><label>ANC Number</label></p>
+                        <p>
+                            <patient field="identifier" identifierTypeId="b0cc255e-ef57-4266-839e-0df7f5099eb9"
+                                     required="true" />
+                        </p>
+                    </td>
+                    <td>
+                        <p><label>REF Number</label></p>
+                        <p>
+                            <obs conceptId="99767" />
+                        </p>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p><label>Gravida</label></p>
+                        <p>
+                            <obs conceptId="5624" />
+                        </p>
+                    </td>
+                    <td>
+                        <p><label>Parity</label></p>
+                        <p>
+                            <obs conceptId="1053" />
+                        </p>
+                    </td>
+                    <td>
+                        <p><label>Weeks of gestation</label></p>
+                        <p>

```

Comment:
@moshonk Global you do not need a <p> tag around both the label and obs, you only need one tag around them. See https://github.com/openmrs/openmrs-module-referenceapplication/blob/master/omod/src/main/webapp/resources/htmlforms/simpleVisitNote.xml